File: 0009 - Palindrome Number/Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0) {
            return false;
        }
        x = abs(x);
        string temp_string = to_string(x);
        string our_string = temp_string;
        reverse(temp_string.begin(), temp_string.end());
        return our_string == temp_string;
    }
};

File: 0009 - Palindrome Number/Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0 : return False
        s = str(x)
        rev_s = ""
        for i in range(len(s) - 1, -1, -1):
            rev_s += s[i]
        return s == rev_s
